File: wedemy_frontend/projects/homepage/src/app/models/detail-class-information.ts
import { DetailClasses } from "./detail-classes";
import { ModuleRegistrations } from "./module-registrations";

export class DetailClassInformation {

    detailClass: DetailClasses = new DetailClasses();
    modules?: ModuleRegistrations[];
    totalHours?: number;
    totalParticipant?: number;
}

File: src/wedemy_backend/elearning/src/main/java/com/lawencon/elearning/helper/DetailClassInformation.java
package com.lawencon.elearning.helper;

import java.util.List;

import com.lawencon.elearning.model.DetailClasses;
import com.lawencon.elearning.model.ModuleRegistrations;

import lombok.Data;

@Data
public class DetailClassInformation {

	private DetailClasses detailClass;
	private List<ModuleRegistrations> modules;
	private Integer totalHours;
	private Integer totalParticipant;

}
